apps/api/src/domain/models/vocabulary-list/queries/vocabulary-list-queries-fetch-by-id.e2e.spec.ts

```diff
@@ -90,108 +72,88 @@ const existingTerm: TermViewModel = TermViewModel.fromDto({
     contributions: [],
 });
 
-const vocabularyListEventStream = new TestEventStream()
-    .andThen<VocabularyListCreated>({
-        type: 'VOCABULARY_LIST_CREATED',
-        payload: {
-            name: vocabularyListName,
-            languageCodeForName: languageCodeForVocabularyListName,
-        },
-    })
-    .andThen<VocabularyListNameTranslated>({
-        type: 'VOCABULARY_LIST_NAME_TRANSLATED',
-    })
-    .andThen<VocabularyListFilterPropertyRegistered>({
-        type: 'VOCABULARY_LIST_PROPERTY_FILTER_REGISTERED',
-    })
-    .andThen<TermAddedToVocabularyList>({
-        type: 'TERM_ADDED_TO_VOCABULARY_LIST',
-        payload: {
-            termId,
-        },
-    })
-    .andThen<TermInVocabularyListAnalyzed>({
-        type: 'TERM_IN_VOCABULARY_LIST_ANALYZED',
-    })
-    .andThen<ResourcePublished>({
-        type: 'RESOURCE_PUBLISHED',
-    });
-
-// TODO check that contributors are joined in
+// TODO check that contributors are joined in- let's do this now
 // const existingContributor = getValidAggregateInstanceForTest(AggregateType.contributor)
```

Comment:
It occurs to me that our new test helper could actually be wrapped inside `getValidAggregateInstanceForTest`!